Index: src/BaseApi/index.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/BaseApi/index.js	(revision 3716d27e486178d5805ce4c1b486fc16425e47b6)
+++ src/BaseApi/index.js	(revision )
@@ -1,8 +1,8 @@
 /**
  *心茹  从这改你自己用的接口加好了
  */
-const ajaxBaseUrl = 'http://192.168.2.9:11110/api/industry/';
-// const ajaxBaseUrl ='http://60.191.78.148:12000'
+// const ajaxBaseUrl = 'http://192.168.2.9:11110/api/industry/';
+const ajaxBaseUrl ='http://60.191.78.148:12001/api/'
 
 export default
 {
Index: src/components/DemoHome.vue
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/components/DemoHome.vue	(revision 3716d27e486178d5805ce4c1b486fc16425e47b6)
+++ src/components/DemoHome.vue	(revision )
@@ -1,6 +1,8 @@
 <template>
   <div class="home">
-    <h2>home</h2>
+    <div>
+      1
+    </div>
   </div>
 </template>
 
@@ -12,7 +14,7 @@
     methods:{
       commit:function () {
         this.$axios({
-          url: api + "product_tasks",
+          url: api + "materials",
           method: "get",
           data: {
             pageSize: 10,
Index: src/App.vue
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/App.vue	(revision 3716d27e486178d5805ce4c1b486fc16425e47b6)
+++ src/App.vue	(revision )
@@ -1,12 +1,23 @@
 <template>
   <div id="app">
-    <demo-header></demo-header>
-    <router-view>11</router-view>
+    <el-container>
+      <el-aside>
+        <demo-header></demo-header>
+      </el-aside>
+      <el-container>
+        <el-header>Header</el-header>
+        <el-main style="padding: 0">
+          <router-view></router-view>
+        </el-main>
+        <el-footer>Footer</el-footer>
+      </el-container>
+    </el-container>
   </div>
 </template>
 
 <script>
   import DemoHeader from '@/page/DemoHeader';
+
   export default {
     name: 'App',
     components: {DemoHeader},
@@ -14,24 +25,20 @@
 </script>
 
 <style>
-  html, body, .el-container {
-    height: 100%;
+  html, body {
     margin: 0px;
-    background-color: aquamarine;
   }
-
   #app {
     position: relative;
     overflow: hidden;
-    height: 100%;
     font-family: 'Avenir', Helvetica, Arial, sans-serif;
     -webkit-font-smoothing: antialiased;
     -moz-osx-font-smoothing: grayscale;
     text-align: center;
     color: #2c3e50;
   }
-
   .el-header, .el-footer {
+    height: 100px !important;
     background-color: #B3C0D1;
     color: #333;
     text-align: center;
@@ -39,11 +46,10 @@
   }
 
   .el-aside {
-    background-color: #D3DCE6;
+    background-color: rgb(84, 92, 100);
     color: #333;
     text-align: center;
     line-height: 200px;
-    height: 100%;
   }
 
   .el-main {
@@ -65,4 +71,5 @@
   .el-container:nth-child(7) .el-aside {
     line-height: 320px;
   }
+
 </style>
Index: src/page/DemoHeader.vue
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/page/DemoHeader.vue	(revision 3716d27e486178d5805ce4c1b486fc16425e47b6)
+++ src/page/DemoHeader.vue	(revision )
@@ -1,19 +1,93 @@
 <template>
+
   <div class="Demo-header">
-    <span class="nav-btn">
-      <router-link to="/home">首页</router-link>
-     </span>
-    <span class="nav-btn">
-      <router-link to="/about">关于</router-link>
-     </span>
-    <span class="nav-btn">
-      <router-link to="/contact">联系方式</router-link>
-     </span>
+    <img src="../../build/logo.png" alt="" width="100%" height="atuo">
+    <el-row class="tac demo-menu">
+      <el-col :span="24">
+        <el-menu
+          default-active="2"
+          class="el-menu-vertical-demo"
+          @open="handleOpen"
+          @close="handleClose"
+          background-color="#545c64"
+          text-color="#fff"
+          active-text-color="#ffd04b">
+          <el-submenu index="1" class="menu-align">
+            <template slot="title">
+              <i class="el-icon-location"></i>
+              <span>物料管理</span>
+            </template>
+            <el-menu-item-group>
+              <el-menu-item index="1-1">
+                <i class="el-icon-setting"></i>
+                <router-link to="/home" width="100%" index="1-1">
+                  首页
+                </router-link>
+              </el-menu-item>
+            </el-menu-item-group>
+          </el-submenu>
+          <el-submenu index="2" class="menu-align">
+            <template slot="title">
+              <i class="el-icon-menu"></i>
+              <span>生产配置</span>
+            </template>
+            <el-menu-item-group>
+              <el-menu-item index="1-1">
+                <i class="el-icon-menu"></i>
+                <router-link to="/about">关于</router-link>
+              </el-menu-item>
+            </el-menu-item-group>
+          </el-submenu>
+          <el-submenu index="3" class="menu-align">
+            <template slot="title">
+              <i class="el-icon-setting"></i>
+              <span>关于我们</span>
+            </template>
+            <el-menu-item-group>
+              <el-menu-item index="1-1">
+                <i class="el-icon-menu"></i>
+                <router-link to="/contact">联系方式</router-link>
+              </el-menu-item>
+            </el-menu-item-group>
+          </el-submenu>
+        </el-menu>
+      </el-col>
+    </el-row>
   </div>
-
 </template>
 <script>
   export default {
     name: 'DemoHeader',
+    methods: {
+      handleOpen(key, keyPath) {
+        console.log(key, keyPath);
+      },
+      handleClose(key, keyPath) {
+        console.log(key, keyPath);
+      }
+    }
   }
 </script>
+<style>
+  .Demo-header {
+    line-height: 0;
+  }
+
+  .demo-menu {
+    width: 100%;
+    height: 100%;
+  }
+
+  .menu-align {
+    text-align: left;
+  }
+
+  .el-submenu__title a {
+    color: #fff;
+    text-decoration: none;
+  }
+
+  .el-menu-item-group ul li a {
+    width: 100%;
+  }
+</style>
